app/controller/source_data_reader.rb

```diff
@@ -1,4 +1,61 @@
 # frozen_string_literal: true
 
+require 'safe_yaml'
+require 'pry'
```

Comment:
use this only for debugging ...

require 'pry'; binding.pry if you wan't to add a breakpoint